docs/source/testing.rst

```diff
@@ -97,41 +97,62 @@ Smoke test:
     $ cd koku/testing
     $ ./run_smoke_tests.sh
     
+=====================
+PDB in Docker
+=====================
 To use pdb while running the koku-server in docker:
-Ensure all migrations are run.
-Stop the server `docker-compose stop koku-server`
-Run the server with service-ports: `docker-compose run —service-ports koku-server`
-Breakpoints will now be stopped in this terminal window.
+    1. Ensure all migrations are run.
+    2. Stop the server `docker-compose stop koku-server`
+    3. Run the server with service-ports: `docker-compose run —service-ports koku- 
+       server`
+
+Breakpoints should now be stopped at, in this terminal window.
+
+To test a specific file using tox, edit this line, for example::
+
+    coverage run {toxinidir}/koku/manage.py test --noinput -v 2 {posargs:masu.test.database}`
```

Comment:
Also I'm not sure if it is documented elsewhere - but it might be worth documenting that you can run unit tests for a single file by providing the path to the file you want to unit test when running tox: 
```
tox -e py36 -- masu.test.external.downloader.azure.test_azure_services.AzureServiceTest
```

Reply:
I agree with @abaiken.  We should not be editing the tox.ini file.  Let's go with her suggestion.